packages/suite-desktop-core/e2e/support/common.ts

```diff
@@ -27,9 +27,28 @@ export const launchSuite = async () => {
     );
 
     const window = await electronApp.firstWindow();
+    const localDataDir = await electronApp.evaluate(({ app }) => app.getPath('userData'));
 
-    return { electronApp, window };
+    return { electronApp, window, localDataDir };
 };
 
 export const waitForDataTestSelector = (window: Page, selector: string, options = {}) =>
     window.waitForSelector(`[data-test="${selector}"]`, options);
+
+export const clickDataTest = (window: Page, selector: string) => {
```

Comment:
We have duplicated of this function in this package in https://github.com/trezor/trezor-suite/blob/test/pw-ReportBug/packages/suite-desktop-core/e2e/tests/electrum.test.ts#L7

Maybe it would be nice to substitute the one in https://github.com/trezor/trezor-suite/blob/test/pw-ReportBug/packages/suite-desktop-core/e2e/tests/electrum.test.ts#L7 with import from here.

Besides I think long-term maybe we will have to create a playwright testing helper package with some of those good utilities. But not now.